src/test/objectstore/test_bluefs.cc

```diff
@@ -828,6 +828,91 @@ TEST(BlueFS, test_tracker_50965) {
   fs.umount();
 }
 
+TEST(BlueFS, test_truncate_stable_53129) {
+
+  ConfSaver conf(g_ceph_context->_conf);
+  conf.SetVal("bluefs_min_flush_size", "65536");
+  conf.ApplyChanges();
+
+  uint64_t size_wal = 1048576 * 64;
+  TempBdev bdev_wal{size_wal};
+  uint64_t size_db = 1048576 * 128;
+  TempBdev bdev_db{size_db};
+  uint64_t size_slow = 1048576 * 256;
+  TempBdev bdev_slow{size_slow};
+
+  BlueFS fs(g_ceph_context);
+  ASSERT_EQ(0, fs.add_block_device(BlueFS::BDEV_WAL,  "wal",  false, 0));
```

Comment:
https://jenkins.ceph.com/job/ceph-pull-requests/84944/consoleFull#-2056816497e840cee4-f4a4-4183-81dd-42855615f2c1

```
2021-11-02T16:53:04.272+0000 7ff4a0fb6180 -1 bdev(0x2355c70 wal) open open got: (2) No such file or directory
../src/test/objectstore/test_bluefs.cc:845: Failure
Expected equality of these values:
  0
  fs.add_block_device(BlueFS::BDEV_WAL, "wal", false, 0)
    Which is: -2
[  FAILED  ] BlueFS.test_truncate_stable_53129 (1 ms)
```

Reply:
@neha-ojha Fixed. It worked locally only because some version of test used fixed "wal", "db" and "slow" files.

Reply:
jenkins test make check

Reply:
jenkins test make check

Reply:
jenkins test make check

Reply:
jenkins test make check

Reply:
jenkins test make check